<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание, просмотр и редактирование маршрутов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignVCenter">
     <widget class="QGroupBox" name="routeGroupBox">
      <property name="title">
       <string>Маршруты</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="1,10,1,1,1">
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Полилайн</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QTextBrowser" name="polylineText"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="openRouteButton">
         <property name="text">
          <string>Открыть</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="addRouteButton">
         <property name="text">
          <string>Создать</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="deleteRouteButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="copyPolylineButton">
         <property name="text">
          <string>Скопировать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTableWidget" name="routeTableWidget">
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Длина, км</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Дата создания</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="addRouteFromPolyButton">
         <property name="text">
          <string>Создать из полилайна</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Доп. операции</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <layout class="QHBoxLayout" name="operationsLayout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="pointGroupBox">
      <property name="title">
       <string>Точки маршрута</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Широта&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="latitudeDoubleSpinBox">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-90.000000000000000</double>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000010000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Долгота</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="longitudeDoubleSpinBox">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000010000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="createWaypointButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="changeWaypointButton">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="deleteWaypointButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QTableWidget" name="pointTableWidget">
         <column>
          <property name="text">
           <string>Широта</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Долгота</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Высота</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="heightDoubleSpinBox">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-12000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>12000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Высота</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="showGraphButton">
         <property name="text">
          <string>Показать график высоты</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Сканировать новые операции</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
